README.md

```diff
@@ -15,12 +15,9 @@ This is the source Dockerfile for the [pantheon-public/build-tools-ci](https://q
   - [Terminus Build Tools Plugin](https://github.com/pantheon-systems/terminus-build-tools-plugin)
   - [Terminus Secrets Plugin](https://github.com/pantheon-systems/terminus-secrets-plugin)
```

Comment:
You also switched the secrets plugin so worth updating it here

Reply:
Ah thanks, I thought very strongly about doing that, but looks like I forgot.